llvm/lib/Support/Z3Solver.cpp

```diff
@@ -6,9 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "llvm/ADT/ScopeExit.h"
```

Comment:
This doesn't look right, I see make_scope_exit in this file. (Note that everything here is behind LLVM_WITH_Z3).

Reply:
Thank you for pointing this out!  I'll remove the `Z3Solver.cpp` bits.